crates/re_types_builder/src/objects.rs

```diff
@@ -43,9 +43,13 @@ impl Objects {
 
         let include_dir_path = include_dir_path.as_ref();
 
+        // TODO(jleibs): Do we need real attributes here?
+        let attrs = Attributes::default();
+
```

Comment:
not sure why you'd build this all the way here..?

Reply:
Ahh, I see one gets built inside of from_raw_enum.  Gone.